app/population/view/daily.njk

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
```

Comment:
These are already defined in [`base.njk`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/098a1522281d1d26953114c2d534c5f80d1bb09e/common/templates/layouts/base.njk#L37) so you shouldn't need to declare them here too.

Reply:
Removed unneded global imports